linux_service/start_moneo_services.sh

```diff
@@ -69,6 +72,10 @@ sudo systemctl start moneo@node_exporter.service
 sudo systemctl start moneo@net_exporter.service
 sudo systemctl start moneo@nvidia_exporter.service
 
+if [ "$PublisherMethod" == "workers" ]; then
+    proc_check false
```

Comment:
Need a `exit 1` here?

Reply:
let me take a look

Reply:
The proc check function will handle the exit conditions in this case

Reply:
Make sense for me, thanks!